Packs/CommonScripts/Scripts/ExtractDomainAndFQDNFromUrlAndEmail/ExtractDomainAndFQDNFromUrlAndEmail.py

```diff
@@ -58,11 +58,11 @@ def get_fqdn(the_input):
 
     if domain:  # Weve removed the filter for "zip" as it is now a valid gTLD by Google
         # get the subdomain using tld.subdomain
-        subdomain = domain.subdomain
```

Comment:
can we do an `if not isinstance(domain, <the correct type>): raise TypeError` instead of ignoring here?

Reply:
Done